README.md

```diff
@@ -89,7 +87,14 @@ hierarchical generative modeling and recursive sampling techniques.
 
 If you want to quickly discover **SDV**, simply click the button below and follow the tutorials!
 
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/sdv-dev/SDV/master?filepath=tutorials)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://bit.ly/sdv-tutorials)
+
+**New! Try the SDV 1.0 Beta**
```

Comment:
Should this also be a third level heading instead of just bolded?

Reply:
As discussed during a call with @npatki this can stay as it is for now.